platform/notifications/core/platform-core-notificationcommands/src/main/java/org/codice/ddf/notifications/commands/SendCommand.java

```diff
@@ -103,7 +109,7 @@ private void sendNotification() throws Exception {
         Event event = new Event(Notification.NOTIFICATION_TOPIC_DOWNLOADS, notification);
 
         // Get OSGi Event Admin service
-        EventAdmin eventAdmin = null;
+        EventAdmin eventAdmin;
```

Comment:
Does the comment on line 111 still make sense after getting rid of OsgiCommandSupport?